source/Halibut.Tests/Support/BackwardsCompatibility/LatestClientAndPreviousServiceVersionBuilder.cs

```diff
@@ -302,13 +303,17 @@ public TService CreateClient<TService>(Action<ServiceEndPoint> modifyServiceEndp
 
             public void Dispose()
             {
-                cancellationTokenSource?.Cancel();
-                Client.Dispose();
-                runningOldHalibutBinary.Dispose();
-                HttpProxy?.Dispose();
-                PortForwarder?.Dispose();
-                disposableCollection.Dispose();
-                cancellationTokenSource?.Dispose();
+                var logger = new SerilogLoggerBuilder().Build().ForContext<ClientAndService>();
+                logger.Information("Dispose called");
+                Action<Exception> logError = e => logger.Warning(e, "Ignoring error in dispose");
+                
+                Try.CatchingError(() => cancellationTokenSource?.Cancel(), logError);
+                Try.CatchingError(Client.Dispose, logError);
```

Comment:
By wrapping each call in a try catch, a failure here wont stop the below methods from being called.